interfaces/builtin/network_manager_test.go

```diff
@@ -170,12 +171,15 @@ func (s *NetworkManagerInterfaceSuite) TestUsedSecuritySystems(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(apparmorSpec.SecurityTags(), HasLen, 2)
 
-	snippet, err := s.iface.ConnectedPlugSnippet(s.plug, s.slot, interfaces.SecurityDBus)
+	dbusSpec := &dbus.Specification{}
```

Comment:
Tests are swapped compared to original here (connected, permanent) but the code is fine.